src/Verlite.CLI/AutoIncrement.cs

```diff
@@ -5,6 +5,7 @@ namespace Verlite.CLI
 {
 	internal enum AutoIncrement
 	{
+		none = -1,
```

Comment:
I would prefer we don't use negatives, rather just plonk it on the end with `none = 3`

Reply:
will change value to 3